drivers/sensor/iis2iclx/iis2iclx.c

```diff
@@ -690,7 +690,7 @@ static int iis2iclx_init(const struct device *dev)
 		.irq_dev_name = DT_INST_GPIO_LABEL(inst, drdy_gpios),	\
 		.irq_pin = DT_INST_GPIO_PIN(inst, drdy_gpios),		\
 		.irq_flags = DT_INST_GPIO_FLAGS(inst, drdy_gpios),	\
-		.int_pin = DT_INST_PROP(inst, int_pin)
+		.drdy_int = DT_INST_PROP(inst, drdy_int)
```

Comment:
I think this should be something like 

    COND_CODE_1(DT_HAS_PROP(DT_INST(inst), drdy_int),
                (DT_INST_PROP(inst, drdy_int)),
                (DT_INST_PROP(inst, int_pin)))

otherwise we're still breaking people.  But that could be part of the discussion of how to handle devicetree deprecations.

Reply:
Yes, correct. I did it too quickly. Sorry.

Reply:
I'm actually planning to go back to original name, even if I think that drdy-int is a better name, but I'll do it next week. In any case I think it would be productive to understand what is the policy in such a case.

Reply:
That's the fastest path to getting this in; #32225 should result in a documented policy.

Reply:
> That's the fastest path to getting this in; #32225 should result in a documented policy.

Sure, just did it!